src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -38,7 +38,7 @@ The browser module is the entry point for all your tests, and it is what interac
 | [browser.context()](/javascript-api/k6-experimental/browser/context)                        | Returns the current [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/).                                                        |
 | [browser.isConnected](/javascript-api/k6-experimental/browser/isconnected) <BWIPT id="453"/>  | Indicates whether the [CDP](https://chromedevtools.github.io/devtools-protocol/) connection to the browser process is active or not.                  |
 | [browser.newContext([options])](/javascript-api/k6-experimental/browser/newcontext/) <BWIPT id="455"/> | Creates and returns a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/).                                                             |
-| [browser.newPage([options])](/javascript-api/k6-experimental/browser/newpage)  <BWIPT id="455"/>       | Creates a new [Page](/javascript-api/k6-experimental/browser/page/) in a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) and returns the page. |
+| [browser.newPage([options])](/javascript-api/k6-experimental/browser/newpage)  <BWIPT id="455"/>       | Creates a new [Page](/javascript-api/k6-experimental/browser/page/) in a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) and returns the page. Pages that have been opened ought to be closed using [`Page.close`](/javascript-api/k6-experimental/browser/page/close/). Pages left open could potentially distort the results of Web Vital metrics. |
```

Comment:
I thought about it and decided to leave it here so that people should be aware of it. It's because people might use `newPage` and they might not bother looking at `Page.close` documentation.